tests/contrib/logging/test_logging.py

```diff
@@ -51,7 +52,10 @@ def capture_function_log(func, fmt=DEFAULT_FORMAT, logger_override=None):
     sh = logging.StreamHandler(out)
 
     try:
-        formatter = logging.Formatter(fmt)
+        if fmt_style:
+            formatter = logging.Formatter(fmt, style=fmt_style)
```

Comment:
nit: might be more maintainable to construct kwargs and expand them here rather than branching

Reply:
```python
kwargs = {}
if fmt_style:
    kwargs["fmt_style"] = fmt_style

formatter = logging.Formatter(fmt, **kwargs)
``

ish?

Reply:
yeah